components/cluster/command/deploy.go

```diff
@@ -90,6 +91,7 @@ func newDeploy() *cobra.Command {
 	cmd.Flags().StringVarP(&opt.IdentityFile, "identity_file", "i", opt.IdentityFile, "The path of the SSH identity file. If specified, public key authentication will be used.")
 	cmd.Flags().BoolVarP(&opt.UsePassword, "password", "p", false, "Use password of target hosts. If specified, password authentication will be used.")
 	cmd.Flags().BoolVarP(&opt.IgnoreConfigCheck, "ignore-config-check", "", opt.IgnoreConfigCheck, "Ignore the config check result")
+	cmd.Flags().BoolVarP(&skipEnable, "skip-enable", "", skipEnable, "Don't start the cluster automaticlly after host restart")
```

Comment:
Prefer to not provide this flag, or find a better name for it.

Reply:
I think it could be just named as `--disable`, like `deploy xxx vx.y.z xxx.yaml --disable`

Reply:
If it's `--disable`, we should execute disable for the cluster (because the default value may be enable in the cluster)

Reply:
So, will we remove this flag?

Reply:
OK